index.html

```diff
@@ -3,10 +3,144 @@
 <head>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <link rel="stylesheet" href="style.css">
-  <title>Document</title>
+  <link rel="stylesheet" href="/css/Nabvar.css">
+  <link rel="stylesheet" href="/css/burgerMenuWindows.css">
+  <link rel="stylesheet" href="/css/dinamicSection.css">
+  <title>CapstoneFernando</title>
 </head>
 <body>
-  
+  <div class="upVar">
+    <ul class="Uplist">
+      <a href=""><img src="/images/facebook.png" class="logFaceAndTwit" srcset=""></a>
+      <a href=""><img src="/images/twitter.png" class="logFaceAndTwit" srcset=""></a>
+      <a href="">English</a>
+      <a href="">My page</a>
+      <a href="">Logout</a>
+    </ul>
+  </div>
+  <header class="header">
+    <div>
+      <nav class="mobile-navbar">
+        <button><img src="./images/mobile_menu 1.png" alt="burgerMenu" onclick ="opneNave();" class="image-burger-menu"></button>
+      </nav>
+    </div>
+    
+    <div class="container-logo">
+      <a href="index.html"><img src="./images/LogoUrban.png" alt= My Logo class="logo"></a>
+    </div>
+      <nav class="navbar">
+        <ul class="nav-list"> 
+          <li>
+            <a href="AboutPage.html">About</a>
+          </li>
+          <li>
+            <a href="#Program">Program</a>
+          </li>
+          <li>
+            <a href="#Join">Join</a>
+          </li>
+          <li>
+          <li>
+            <a href="#Sponsor">Sponsor</a>
+          </li>
+          <li>
+            <a href="#News">News</a>
+          </li>
+            <a href="#" class="header_button" id="header_button">UC Campaigns</a>
+          </li>
+        </ul>
```

Comment:
- [ ] Would you mind adjusting the alignment of your links within the navbar? It seems they're appearing a bit differently compared to the template. If you could align them to the right and double-check the spacing, that would be fantastic!

   ### Your design:
   ![image](https://github.com/Fernando426/CapstoneN01/assets/107952724/9e5746b4-c5fd-4837-b177-89b93721a6ae)

   ### Behance design:
   ![image](https://github.com/Fernando426/CapstoneN01/assets/107952724/8c6dde45-ed45-4b93-99dc-4cc3dad7f512)